components/Form/Toggle.tsx

```diff
@@ -0,0 +1,68 @@
+import { css, StyleSheet } from "aphrodite";
+import { ReactElement, useState } from "react";
+import colors from "~/config/themes/colors";
+
+type ToggleOption = {
+  label: string;
+  value: any;
+  selected: any;
+}
+
+type Props = {
+  options: Array<ToggleOption>;
+  onSelect: Function;
+  selected: any;
+};
+
+function Toggle({
+  options,
+  onSelect,
+  selected,
+}: Props): ReactElement {
+
+  const handleSelect = (opt:ToggleOption) => {
+    onSelect(opt);
+  }
```

Comment:
looks like this func isn't doing anything. 
Let's remove it and if we see the need to have its own handleSelect function, we can add back in(?)

Reply:
yeah, I had it do something else at some point. I'll clean it up